content/docs/reference/config/installation-options/index.md

```diff
@@ -156,9 +156,6 @@ To customize Istio install using Helm, use the `--set <key>=<value>` option in H
 | `global.proxy.includeInboundPorts` | `"*"` |  |
```

Comment:
fwiw No need to modify this file - it is autogenerated.  But thanks anyway - we can autogenerate prior to 1.1 with the 1.1 binaries.  Cheers :)

Reply:
Yeah, but I didn't find the make target of how to update this file.